frontend/packages/console-app/locales/en/console-app.json

```diff
@@ -271,6 +271,17 @@
   "Access review rules": "Access review rules",
   "Incompatible file type": "Incompatible file type",
   "{{fileName}} cannot be uploaded. Only {{fileExtensions}} files are supported currently. Try another file.": "{{fileName}} cannot be uploaded. Only {{fileExtensions}} files are supported currently. Try another file.",
+  "Access our new quick starts where you can learn more about creating or deploying an application using OpenShift Developer Console. You can also restart this tour anytime here.": "Access our new quick starts where you can learn more about creating or deploying an application using OpenShift Developer Console. You can also restart this tour anytime here.",
+  "Set your individual console preferences including default views, language, import settings, and more.": "Set your individual console preferences including default views, language, import settings, and more.",
+  "Stay up-to-date with everything OpenShift on our <2>blog</2> or continue to learn more in our <6>documentation</6>.": "Stay up-to-date with everything OpenShift on our <2>blog</2> or continue to learn more in our <6>documentation</6>.",
+  "Welcome to the Developer Perspective!": "Welcome to the Developer Perspective!",
+  "Our new update with OpenShift 4.19 gives a more modern look to help enhance your experience and streamline your workflow, shuch as improved navigation and infrastructure. Want us to show your around?": "Our new update with OpenShift 4.19 gives a more modern look to help enhance your experience and streamline your workflow, shuch as improved navigation and infrastructure. Want us to show your around?",
```

Comment:
1. such spelling is wrong. shuch

2. Want us to show your around? is this Want us to show you around?